src/containers/Home/HomeHero/hello-world.ts

```diff
@@ -6,7 +6,7 @@ const HelloWorld: {[isoLangCode: string]: string} = {
   av: 'Ворчӏами дуниял',
   az: 'Salam Dünya',
   be: 'Прывітанне свет',
-  bn: 'স্বাগতম বিশ্ব',
+  bn: 'ওহে বিশ্ব',
```

Comment:
This shows Oh World in google. I hope it's correct in native language ?

Reply:
I checked the dictionary and the google translate. If you search using "Hello World in Bengli" it will translate it to 'ওহে বিশ্ব'. The English to Bengali dictionary also translates to this.
BTW, Bengali is my native.